python/example.py

```diff
@@ -26,13 +15,13 @@
 print("NUMBER OF PARAMETERS: param_num")
 for tp in [True, False]:
     for gq in [True, False]:
-        D = model.param_num(include_tp = tp, include_gq = gq)
+        D = model.param_num(include_tp=tp, include_gq=gq)
         print(f"tp = {tp:b}, gq = {gq:b}, number of parameters = {D}\n")
 
 print("PARAMETER NAMES: param_names")
 for tp in [True, False]:
     for gq in [True, False]:
-        names = model.param_names(include_tp = tp, include_gq = gq)
+        names = model.param_names(include_tp=tp, include_gq=gq)
```

Comment:
[question]
Sorry---forgot to run this all through `black`!  Is there an easy way to automate that with CI?

Reply:
The preferred way in the python world is usually a tool called [pre-commit](https://pre-commit.com/). We could set that up here if we wanted, or we could try to do something like that Stan's C++ repositories do and push a commit to your branch to fix it automatically. I don't love those, since you end up with a lot of merge commits and bot commits in the history.